src/snapred/backend/recipe/algorithm/DetectorPeakPredictor.py

```diff
@@ -71,16 +75,16 @@ def PyExec(self) -> None:
             for d in dList:
                 # beta terms
                 beta_T = beta_0 + beta_1 / d**4  # GSAS-I beta
-                beta_d = 505.548 * L * np.sin(tTheta / 2) * beta_T  # converted to d-space
+                beta_d = self.BETA_D * L * np.sin(tTheta / 2) * beta_T  # converted to d-space
 
-                fwhm = 2.35 * delDoD * d
+                fwhm = self.FWHM * delDoD * d
```

Comment:
I think this one is already defined in `calibration.parameters.FWHMMultiplier`

Reply:
I believe this is different. 2.35 is actually $2* \sqrt{2*\log_{e} 2}$ to convert to fwhm from gaussian